resources/js/Pages/OfficeLife/Index.vue

```diff
@@ -165,24 +166,17 @@ export default {
   data() {
     return {
       localPlans: [],
-      form: {
-        quantity: 0,
-      },
     };
   },
 
   methods: {
     checkPrice(plan) {
-      this.form.quantity = plan.quantity;
-
       axios
-        .post(plan.url.price, this.form)
+        .post(route('officelife.price', { plan: plan.id }), { quantity: plan.quantity })
```

Comment:
I don't like this, but I won't fight it on this project.
In my opinion we shouldn't call route names in the views directly. This knowledge should come from the backend.

Reply:
There are a lots of `route` calls in vue, I don't understand your point.